SR2019/2019-01-26-competition-date-announcement.md

```diff
@@ -0,0 +1,24 @@
+---
+to: Student Robotics 2019
+subject: Confirming the dates for Student Robotics 2019
+---
+
+Hello all,
+
+As some eagle-eyed members of your team may have noticed on [the forum](https://studentrobotics.org/forum/viewtopic.php?f=9&t=35), today's the day we confirm the dates for Student Robotics 2019!
+
+The competition will be held from *Saturday 6th April* until *Sunday 7th April*. The competition will be held at the University of Southampton's Highfield campus, so please plan accommodation / transport accordingly. The exact start/end times are to be confirmed, but doors open 9am, and we aim to finish around 5pm.
+
+[Competition details](https://studentrobotics.org/events/sr2019/competition/)
+
+## Tech Days
+
+There are just two remaining tech days between now and the competition. These are a great chance to spend a full day testing on your kit, and get direct support from us. The remaining tech days are:
```

Comment:
Not sure about this wording - it sounds more like kickstart? Perhaps: "These are a great chance to spend a full day working on your robot, and getting direct support from us"?